lib/mobility-core/src/Kernel/External/SMS/Interface/Types.hs

```diff
@@ -69,3 +80,18 @@ sendOtpResToText = \case
   Fail -> "Fail"
   Pending -> "Pending"
   UnknownError -> "unknown request"
+
+data SMSType = Transactional | Promotional | TransactionalOptIn
+  deriving (Generic, FromJSON, ToJSON, Show, Eq)
+
+data PriorityType = Normal | High
+  deriving (Generic, FromJSON, ToJSON, Show, Eq)
+
+instance ToHttpApiData SMSType where
```

Comment:
why not just use toLower?

Reply:
transaction_opt_in this type also exists so we can't directly use toLower

Reply:
Why can't you do for this too?

Reply:
> why not just use toLower?
toLower is not suitable because we need convert camel to snake also

Reply:
Then `aesonPrefix snakeCase` can be used?